Meadowlark-js/tests/http/local.ownership-tests.http

```diff
@@ -2,8 +2,8 @@
 # Be sure to load all descriptors into the database first.
 
 
-@auth_header_1=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InN1cGVyLWdyZWF0LVNJUyIsImp0aSI6IjNkNTliNzVmLWE3NjItNGJhYS05MTE2LTE5YzgyZmRmOGRlMyIsImlhdCI6MTYzNjU2MjA2MCwiZXhwIjozODQ1NTQ4ODgxfQ.WF5ABFZAvNsLDZktwa8VxDR846fGptRXJObtQitbL8I
-@auth_header_2=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InNtYWxsLXRvd24tc2lzIiwianRpIjoiMGQwZWZmYTctYWExZi00N2E3LTk4NzktNDdhOGJkOWMzMWJiIiwiaWF0IjoxNjM2NTYyMDc5LCJleHAiOjM4NDU1NDg4ODF9.2QGyvx9flXdM9wjEHX_VPmeacCI3nm8WcV-T5AWxeWo
+@auth_header_1=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJzdWIiOiJzdXBlci1ncmVhdC1TSVMiLCJqdGkiOiIyNGMyMDIyMC1mMjg1LTRlNjMtYThmNC1iMmRkYzM5MzAwMDQiLCJpYXQiOjE2NTg1MjM0MDgsImV4cCI6MjY1ODUyNzAwOH0.S2C44HoFMr5ZYSUkbZoAV2pz7Z1Xi40yynH17GOBVoY
+@auth_header_2=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJzdWIiOiJzdXBlci1ncmVhdC1TSVMiLCJqdGkiOiI4MjVkNDRiOC1hODExLTRjZmYtYmVmOC1jYzhhNjYxNjliZTAiLCJpYXQiOjE2NTg1MjQyMzUsImV4cCI6MjY1ODUyNzgzNX0.DR9-ixkAs-EUIsoRjYYGY-NLal2o8wrDYYLcKFfb3_I
```

Comment:
@stevenarnold-ed-fi these two tokens are for the same SIS ("super-great-sis"), which invalidates the tests in this file. For example, the post on line 23 no longer fails.

I generated a new long-lived token for the second set of credentials:

```
eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJzdWIiOiJzdXBlci1ncmVhdC1TSVMiLCJqdGkiOiIyMDJhY2Y0My0xZDNkLTQ5YzMtYTZiMy1lNzIxZmQ5ZjE3NGYiLCJpYXQiOjE2NTg3NzQ3NzUsImV4cCI6MjUyNzQ1OTIwMH0.X4gV3613Xq3oDwGh7RSQfbZVPhc13NuYUpibp1_rOxc
```

Even with that in place, the tests are not failing properly. For example, the first token creates an `EducationContents` resource, and the second token should not be able to access it.

I recommend also adding a third set of credentials in `HardcodedCredentials.ts`, just for host provider. Maybe...

```
export const client3: HardcodedCredential = {
  key: 'host_key',
  secret: 'host_secret',
  vendor: 'hosting-provider',
};
```

Then automatically grant this set of credentials, when received, the correct Role of "Host".